charms/istio-gateway/config.yaml

```diff
@@ -8,3 +8,7 @@ options:
     type: string
     description: |
       Type of service for the ingress gateway out of: 'ClusterIP', 'LoadBalancer', or 'NodePort'.
+  proxy-image:
+    default: 'docker.io/istio/proxyv2:1.17.3'
```

Comment:
Same here with the [other PR](https://github.com/canonical/istio-operators/pull/321/files/d686f6e4e977734ff2050f17bbef343d5b83992d#r1311898117), where do we get this image from?

Reply:
FIled separate issue to track it https://github.com/canonical/istio-operators/issues/324